packages/datatrak-web/src/views/AccountSettingsPage/AccountSettingsSection.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { Paper, Typography } from '@material-ui/core';
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+import { Button } from '../../components';
+
+const Wrapper = styled(Paper).attrs({
+  elevation: 0,
+  component: 'section',
+})`
+  padding: 0.8rem;
+  margin-top: 1.5rem;
+  display: flex;
+  flex-direction: column;
+  ${({ theme }) => theme.breakpoints.up('md')} {
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 2.5rem;
+  }
+`;
+
+const Title = styled(Typography).attrs({
+  variant: 'h2',
+})`
+  font-size: 1rem;
+  margin-bottom: 0.6rem;
+  font-weight: ${({ theme }) => theme.typography.fontWeightMedium};
+`;
+
+const Column = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-bottom: 1rem;
+  &:not(:last-child) {
+    padding-right: 2rem;
+  }
+  &:first-child {
+    margin-bottom: 1.9rem;
+  }
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  ${({ theme }) => theme.breakpoints.up('md')} {
+    width: 28.5%;
+    margin-bottom: 0;
+    &:first-child {
+      width: 43%;
+      margin-bottom: 0;
+      p {
+        max-width: 22rem;
+      }
+    }
+    &:last-child {
+      justify-content: flex-end;
+    }
+  }
+`;
+
+interface AccountSettingsSectionProps {
+  title?: ReactNode;
+  description?: string;
+  leftColumn?: ReactNode;
+  centerColumn?: ReactNode;
+  rightColumn?: ReactNode;
+  button?: {
+    label: string;
+    onClick: () => void;
+    tooltip?: string;
+    disabled?: boolean;
+  };
+}
+
+export const AccountSettingsSection = ({
+  title,
+  description,
+  leftColumn,
+  centerColumn,
+  rightColumn,
+  button,
+}: AccountSettingsSectionProps) => {
+  return (
+    <Wrapper>
```

Comment:
I think this layout component is a bit too opinionated to cover all the sections because:
- the three other sections will be a form that goes across columns 2 and 3
- the button should be a form submit button in the other sections
- the title of the Request country access section includes a custom button

I'm not sure the best solution but perhaps the component could simply be:
- the main layout with 2 columns (one third and then two thirds)
- title prop which could either be text or a custom component
- description

But I don't mind if you have other ideas about how to solve it
- the personal details